src/windshaft/filters/bounding-box.js

```diff
@@ -1,8 +1,29 @@
+var _ = require('underscore');
 var Model = require('../../core/model');
+var BOUNDING_BOX_FILTER_WAIT = 300;
 
 module.exports = Model.extend({
-  initialize: function (bounds) {
-    this.setBounds(bounds);
+  initialize: function (map) {
+    if (!map) {
+      throw new TypeError('Bounding box filter needs a map to get instantiated.');
+    }
+
+    this._map = map;
+    this.setBounds(this._map.getViewBounds());
+    this._initBinds();
+  },
+
+  _initBinds: function () {
+    this.listenTo(this._map, 'change:view_bounds_ne change:center change:zoom', _.debounce(this._boundsChanged, BOUNDING_BOX_FILTER_WAIT));
+  },
+
+  _stopBinds: function () {
+    this.stopListening(this._map, 'change:view_bounds_ne change:center change:zoom');
+  },
+
+  _boundsChanged: function () {
+    this.setBounds(this._map.getViewBounds());
```

Comment:
we should tests the integration between the map and the boundFilter to prevent bugs if the `map.getViewBounds()` is changed.